config/cscs.py

```diff
@@ -95,6 +100,14 @@ class ReframeSettings:
 
                     'gpu': {
                         'scheduler': 'nativeslurm',
+                        'container_platforms': {
+                            'ShifterNG': {
+                                'modules': ['shifter-ng']
+                            },
+                            'Singularity': {
+                                'modules': ['singularity']
```

Comment:
@rsarm @teojgo Do you think we will need other config parameters, e.g., default mounts, environment variables, etc.?

Reply:
For the moment I think we are fine. If we plan to support image building in the future we can add more.

Reply:
So Singularity works fine out of the box for MPI and GPU programs, as soon as you load the module?

Reply:
We have a module wrapper for this. In the singularity case you can add also environment variables to the test or add the module wrapper to the modules.

Reply:
Ok, my question then is would it make sense for this wrapper to be a partition configuration parameter, i.e., available by default to all checks that are using Singularity?

Reply:
The wrapper module we make available on both gpu/mc partitions (for dom it is missing at the moment but for daint it is there). I don't know what would make more sense.